docs/modules/user-guide/partials/proc_updating-the-devfile-library-schema.adoc

```diff
@@ -0,0 +1,13 @@
+[id="proc_updating-the-devfile-library-schema_{context}"]
+= Updating the devfile library schema
+
+Update the schema version to ensure that you develop with the latest code from devfile properties:
+
+. To fetch the latest `github.com/devfile/api` Go module and to update the schema saved under `pkg/devfile/parser/data`, run, `./scripts/updateApi.sh`.
+. Update the devile schema version by running, `./updateAPI.sh 2.x.x`.
+If you run the script with no arguments, it updates the latest devfile version.
```

Comment:
Maybe consider putting this info in a "Note" admonition? It feels like it's just "additional guidance or advice" per the SSG's recommendations on Note admonitions.

Reply:
```suggestion
Running the script this way, with no arguments, updates the latest devfile version.
```
Can we rephrase this to reflect that the command does not contain any arguments?